File: stress/t/int_gcd.c
#include <stdio.h>
#include <stdint.h>
int main(void){
  int64_t a=1071,b=462,t;
  while(b!=0){ t=a%b; a=b; b=t; }
  printf("gcd="); printf("%ld",(long)a); printf("\n");
  a=123456; b=7890;
  while(b!=0){ t=a%b; a=b; b=t; }
  printf("gcd2="); printf("%ld",(long)a); printf("\n");
  return 0;
}

File: stress/t/int_gcd.f
      PROGRAM INTGCD
      INTEGER*8 A,B,T
      A=1071
      B=462
   10 IF (B.EQ.0) GOTO 20
      T=MOD(A,B)
      A=B
      B=T
      GOTO 10
   20 CONTINUE
      WRITE(*,'(A,I0)') 'gcd=',A
      A=123456
      B=7890
   30 IF (B.EQ.0) GOTO 40
      T=MOD(A,B)
      A=B
      B=T
      GOTO 30
   40 CONTINUE
      WRITE(*,'(A,I0)') 'gcd2=',A
      END
